pilot/pkg/kube/inject/testdata/webhook/TestWebhookInject_https_probe_rewrite_template.yaml

```diff
@@ -0,0 +1,23 @@
+rewriteAppHTTPProbe: true
+initContainers:
+- name: istio-init
+  image: example.com/init:latest
+containers:
+- name: istio-proxy
+  image: example.com/proxy:latest
+  args:
+    - --statusPort
+    - 15020
+imagePullSecrets:
+- name: istio-image-pull-secrets
+volumes:
+- emptyDir:
+    medium: Memory
+  name: istio-envoy
+- name: istio-certs
+  secret:
+    [[ if eq .Spec.ServiceAccountName "" -]]
```

Comment:
Why we need this template `[[ if serviceaccount]` in the sidecar injector template? Can we use an existing one? I think all you need is `rewriteAppProbe = true`. Right?

Reply:
yes, we can. I just copied this from an existing (TestWebhookInject_http_probe_rewrite_template.yaml), and modified the probe part.